services/Postmates.mjs

```diff
@@ -280,5 +282,311 @@ class Postmates extends Service {
+        "accept-language": "en-US,en;q=0.9",
+        "content-type": "application/json",
+        dnt: "1",
+        origin: "https://postmates.com",
+        "sec-ch-ua":
+          '"Chromium";v="110", "Not A(Brand";v="24", "Google Chrome";vx ="110"',
+        "sec-ch-ua-mobile": "?0",
+        "sec-ch-ua-platform": '"Windows"',
+        "sec-fetch-dest": "empty",
+        "sec-fetch-mode": "cors",
+        "sec-fetch-site": "same-origin",
+        "user-agent":
+          "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36",
+        "x-csrf-token": "x",
+      },
+      body: JSON.stringify({
+        itemRequestType: "ITEM",
+        storeUuid: storeID,
+        sectionUuid: sectionID,
+        subsectionUuid: subsectionID,
+        menuItemUuid: itemID,
+      }),
+    });
+    if (res.ok) {
+      return {
+        data: await res.json(),
+      };
+    } else {
+      throw new HTTPResponseError(res);
+    }
+  }
+
+  /* Add Item to Postmates Cart
+   *@param draftOrderID {Object} is obtained from createCart().
+   *@param cartID {Object} is obtained from createCart().
+   @param itemID->image is obtained from getMenu() and getItemDetails().
+   */
+
+  async addToCart({
+    draftOrderID,
+    cartID,
+    itemID,
+    storeID,
+    sectionID,
+    subsectionID,
+    priceAsCents,
+    title,
+    itemQuantity,
+    customizations,
+    image,
+  }) {
+    const generatedItemUUID = uuidv4();
+    const res = await fetch(
+      "https://postmates.com/api/addItemsToDraftOrderV2",
+      {
+        method: "POST",
+        headers: {
+          authority: "postmates.com",
+          accept: "*/*",
+          "accept-language": "en-US,en;q=0.9",
+          "content-type": "application/json",
+          dnt: "1",
+          origin: "https://postmates.com",
+          "user-agent":
+            "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36",
+          "x-csrf-token": "x",
+        },
+        body: JSON.stringify({
+          draftOrderUUID: draftOrderID,
+          cartUUID: cartID,
+          items: [
+            {
+              uuid: itemID,
+              shoppingCartItemUuid: generatedItemUUID,
+              storeUuid: storeID,
+              sectionUuid: sectionID,
+              subsectionUuid: subsectionID,
+              price: priceAsCents,
+              title: title,
+              quantity: itemQuantity,
+              customizations: customizations,
+              imageURL: image,
+              specialInstructions: "",
+              itemId: null,
+            },
+          ],
+          shouldUpdateDraftOrderMetadata: false,
+          storeUUID: storeID,
+        }),
+      }
+    );
+    if (res.ok) {
+      return {
+        data: await res.json(),
+      };
+    } else {
+      throw new HTTPResponseError(res);
+    }
+  }
+
+  async getFee({ draftOrderID }, cookies) {
```

Comment:
Let's add some function docs (params, descriptions, return vals) to this method.

Reply:
Fixed